src/components/state/loading.scss

```diff
@@ -0,0 +1,38 @@
+.loading {
+  animation: 0s linear 750ms forwards delayedShow;
+  visibility: hidden;
+
+  position: absolute;
+  inset: 0;
+  margin-bottom: 47px;
```

Comment:
We need to change this margin according how high the control toolbar is. In full screen mode, there is a gap between overlay and toolbar.

Reply:
what about moving the component inside of` xm-screen`?

Reply:
This sound reasonable, but we only render the screen component [if there are two videos side by side](https://github.com/openHPI/xikolo-video-player/blob/main/src/components/player/player.tsx#L757). The naming `xm-screen` is somewhat misleading.

Reply:
right. Let's adjust this margin then or add a fix height for the controls.

Reply:
I've adjusted the margin [here](https://github.com/openHPI/xikolo-video-player/pull/41/commits/87fe619b4776cfd1348cdbe1f8857f18b3ded2c8). For that, the status needs to be passed to the component.